index.js

```diff
@@ -316,6 +316,83 @@
     return x;
   });
 
+  //. ### Composition
+
+  //# compose :: (b -> c) -> (a -> b) -> a -> c
+  //.
+  //. Takes two functions assumed to be unary and a value of any type,
+  //. and returns the result of applying the first function to the result
+  //. of applying the second function to the given value.
+  //.
+  //. In general terms, `compose` performs right-to-left composition of two
+  //. unary functions.
+  //.
+  //. See also [`pipe`](#pipe).
+  //.
+  //. ```javascript
+  //. > S.compose(Math.sqrt, R.inc)(99)
+  //. 10
+  //. ```
+  var compose = S.compose =
+  def('compose', [Function, Function, a], function(f, g, x) {
+    return f(g(x));
+  });
+
+  //# pipe :: [(a -> b), (b -> c), ..., (m -> n)] -> a -> n
+  //.
+  //. Takes a list of functions assumed to be unary and a value of any type,
+  //. and returns the result of applying the sequence of transformations to
+  //. the initial value.
+  //.
+  //. In general terms, `pipe` performs left-to-right composition of a list
+  //. of functions. `pipe([f, g, h], x)` is equivalent to `h(g(f(x)))`.
+  //.
+  //. See also [`meld`](#meld).
+  //.
+  //. ```javascript
+  //. > S.pipe([R.inc, Math.sqrt, R.dec])(99)
+  //. 9
+  //. ```
+  var pipe = S.pipe = def('pipe', [List, a], function(fs, x) {
+    return R.reduceRight(compose, R.identity, fs)(x);
+  });
+
+  //# meld :: [** -> *] -> (* -> * -> ... -> *)
+  //.
+  //. Takes a list of non-nullary functions and returns a curried function
+  //. whose arity is one greater than the sum of the arities of the given
+  //. functions less the number of functions.
+  //.
+  //. The behaviour of `meld` is best conveyed diagrammatically. The following
+  //. diagram depicts the "melding" of binary functions `f` and `g`:
+  //.
+  //.               +-------+
+  //.     --- a --->|       |
+  //.               |   f   |                +-------+
+  //.     --- b --->|       |--- f(a, b) --->|       |
+  //.               +-------+                |   g   |
+  //.     --- c ---------------------------->|       |--- g(f(a, b), c) --->
+  //.                                        +-------+
+  //.
+  //. See also [`pipe`](#pipe).
+  //.
+  //. ```javascript
+  //. > S.meld([Math.pow, R.multiply, R.subtract])(3, 4, 5, 6)
```

Comment:
Using two binary functions in the examples to match the diagram is a great idea! Do you think this is suitable?

```javascript
//. > S.meld([Math.pow, R.subtract])(3, 4, 5)
//. 76
```

I'll welcome suggestions for better examples. :)